applications/main/subghz/subghz_cli.c

```diff
@@ -519,6 +520,7 @@ static void subghz_cli_command_print_usage() {
         printf("  debug cmd:\r\n");
         printf("\ttx_carrier <frequency:in Hz>\t - Transmit carrier\r\n");
         printf("\trx_carrier <frequency:in Hz>\t - Receive carrier\r\n");
+        printf("\tcalibrate <frequency offset:in ppm>\t - Calibrate crystal\n");
```

Comment:
Calibrate sounds like an action, but it's not. See furi_hal_subghz comment for more details.

Reply:
How do you feel about `set_offset`?